src/Ng/Program.cs

```diff
@@ -27,22 +27,22 @@ public static void Main(string[] args)
                 Debugger.Launch();
             }
 
-            // Get arguments
-            var arguments = CommandHelpers.GetArguments(args, 0);
-
-            // Configure ApplicationInsights
-            ApplicationInsights.Initialize(CommandHelpers.GetApplicationInsightsInstrumentationKey(arguments));
-
             // Create an ILoggerFactory
             var loggerConfiguration = LoggingSetup.CreateDefaultLoggerConfiguration(withConsoleLogger: true);
             var loggerFactory = LoggingSetup.CreateLoggerFactory(loggerConfiguration);
 
             // Create a logger that is scoped to this class (only)
             _logger = loggerFactory.CreateLogger<Program>();
 
-            var cancellationTokenSource = new CancellationTokenSource();
             try
             {
+                // Get arguments
+                var arguments = CommandHelpers.GetArguments(args, 0);
+
+                // Configure ApplicationInsights
+                ApplicationInsights.Initialize(CommandHelpers.GetApplicationInsightsInstrumentationKey(arguments));
```

Comment:
>Initialize [](start = 36, length = 10)

I suspect that this line has to run before CreateLoggerFactory. U can verify this by testing the code and using this AI app: [https://ms.portal.azure.com/?resourceMenuPerf=true#resource/subscriptions/fa8b3229-dc0f-4633-a285-ad597076921d/resourceGroups/NugetDevGroup/providers/microsoft.insights/components/StageManager/overview](https://ms.portal.azure.com/?resourceMenuPerf=true#resource/subscriptions/fa8b3229-dc0f-4633-a285-ad597076921d/resourceGroups/NugetDevGroup/providers/microsoft.insights/components/StageManager/overview)